src/dtx/dtx_rpc.c

```diff
@@ -312,9 +312,6 @@ dtx_req_list_send(struct dtx_req_args *dra, crt_opcode_t opc, d_list_t *head,
 				dra->dra_future = ABT_FUTURE_NULL;
 				return rc;
 			}
-
-			drr->drr_result = rc;
```

Comment:
so we just continue the loop when this happens and ignore the return code from dtx_req_send ?

Reply:
Yes, as described in the source code:
                        /* If the first sub-RPC failed, then break, otherwise
                         * other remote replicas may have already received the
                         * RPC and executed it, so have to go ahead.
                         */

Means that some non-leader may have already received the RPC and executed it, so we have to continue even if some other non-leaders failed to do that.